examples/reference/custom_components/PreactComponent.md

```diff
@@ -0,0 +1,401 @@
+# `PreactComponent`
```

Comment:
Are we sure we want to expose and support the `PreactComponent`? It is extra work and maybe its better to focus on making the perfect `ReactComponent`?

Reply:
Yeah, on the fence about this. One drawback of the ReactComponent is that it fetches React from a CDN while the PreactComponent uses our bundled Preact used for rendering `ReactiveHTML`.